docs/earthfile/earthfile.md

```diff
@@ -1018,6 +1066,13 @@ If an `IMPORT` is defined in the `base` target of the Earthfile, then it becomes
 
 For more information see the [target, artifact and command references guide](../guides/target-ref.md).
 
+#### Options
+
+##### `--allow-privileged`
+
+Same as [`FROM --allow-privileged`](#allow-privileged).
```

Comment:
I would expand a little more here, given that this is the `IMPORT` command. We should say that it allows any mention of the alias to use privileged.

Reply:
That's a good call, how does it sound now?